test/mw_config_data.js

```diff
@@ -0,0 +1,97 @@
+// Includes only what's needed for morebits, util, and Title
+mw.config.set({
+	// Needed by Morebits
+	pageTriageNamespaces: [0, 2, 118],
+	wgArticleId: 18866,
+	wgIsRedirect: false,
+	wgPageName: 'Macbeth',
+	wgRestrictionEdit: ['autoconfirmed'],
+	wgUserGroups: ['interface-admin', 'oversight', 'patroller', 'reviewer', 'sysop', '*', 'user', 'autoconfirmed'],
+	wgUserName: 'Amorymeltzer',
+
+	// Needed by mw.util
+	wgScript: '/w/index.php',
+	wgArticlePath: '/wiki/$1',
+	wgScriptPath: '/w',
+
+	// Needed by mw.Title
+	wgFormattedNamespaces: {
+                '-2': 'Media',
```

Comment:
indentations are messed up in this file.

Reply:
s/spaces/tabs/